lib/go/minifier/minifier.go

```diff
@@ -0,0 +1,79 @@
+package main
+
+import (
+	"bufio"
+	"flag"
+	"io"
+	"log"
+	"os"
+	"strings"
+)
+
+const commentsPrefix  = "//"
```

Comment:
Consider running gofmt on all files

```suggestion
const commentsPrefix = "//"
```